systemtest/loadgen/eventhandler/handler.go

```diff
@@ -183,6 +185,39 @@ func (h *Handler) WarmUpServer(ctx context.Context) error {
 	}
 }
 
+// Clone creates a new instance of the Handler, copying the batches contents
+// to a new slice.
+func (h *Handler) Clone() (*Handler, error) {
```

Comment:
Perhaps a stupid question: is it needed to clone, or can we reset the batches in the Handler.Reset method? I think resetting to the initial state after using the handler would fit better with the pool pattern.

Reply:
I think I answered this question above.